app/models/metric/ci_mixin/capture.rb

```diff
@@ -205,7 +206,16 @@ def perf_capture(interval_name, start_time = nil, end_time = nil)
                                              :start_range          => start_range
                                             )
       end
-      perf_process(interval_name, start_range, end_range, counters, counter_values)
+
+      # Convert to format allowing to send multiple resources at once
+      counters_data = {
```

Comment:
@agrare so something like this can easily go through a queue, having multiple resources inside

Reply:
Agreed this will be perfect to queue `perf_process` for the metrics_processor_worker

Reply:
cool :+1: 

For 5.0, I would like to move to the generic Persistor format, so we can reuse inventory Persistors (and using grouping, we should be able to make generic worker a dedicated worker?)

Reply:
We are going over to using the queue before 5.0 - so would be nice to go over to the generic `Persistor` format within the next week or so here

Reply:
@kbrock for 4.6, I think we will just keep this format. Generic Persistors are 5.0 thing, right @agrare ?

Reply:
I think this format is fine since we're going to be invoking `perf_process` directly in the `metrics_processor_worker` not some generic persister.